modules/ocf_docker/manifests/deckschrubber.pp

```diff
@@ -0,0 +1,20 @@
+class ocf_docker::deckschrubber {
+  package { 'deckschrubber': }
+
+  file { ['/opt/docker', '/opt/docker/registry']:
+    ensure => directory,
+    mode   => '0755',
+  } ->
+  file { '/opt/docker/registry/garbage-collect':
+    source  => 'puppet:///modules/ocf_docker/garbage-collect',
+    mode    => '0755',
+    require => Package['deckschrubber'],
+  } ->
+  cron { 'registry-gc':
+    command  => '/opt/docker/registry/garbage-collect',
+    user     => 'root',
+    monthday => 1,
```

Comment:
weekly seems good to me, the only host on which this operates is biohazard anyways

Reply:
I picked one month to minimize the downtime the registry has, but once per week sounds ok to me as well.

Reply:
I'd agree with once per week, every day seems too frequent because it won't even clean up much after that long and it takes the registry out of service, and every month seems too infrequent since one failure could mean a whole month of images piles up.

Reply:
On the other hand though, in the past we have only needed to clean the docker registry once every several months. I personally would prefer cleaning on a monthly basis since it's less frequent downtime.